pkg/minishift/clusterup/clusterup.go

```diff
@@ -64,14 +65,39 @@ type ClusterUpConfig struct {
 func ClusterUp(config *ClusterUpConfig, clusterUpParams map[string]string, runner util.Runner) error {
 	cmdArgs := []string{"cluster", "up", "--use-existing-config"}
 
+	// Deal with extra flags
+	var extraFlags []string
+	if val, ok := clusterUpParams[startFlags.ExtraClusterUpFlags.Name]; ok {
+		extraFlags = strings.Fields(val)
+		delete(clusterUpParams, startFlags.ExtraClusterUpFlags.Name)
+	}
+
+	// Check if clusterUp flags are supported
 	for key, value := range clusterUpParams {
 		if !oc.SupportFlag(key, config.OcPath, runner) {
+			println(key)
 			return errors.New(fmt.Sprintf("Flag %s is not supported for oc version %s. Use 'openshift-version' flag to select a different version of OpenShift.", key, config.OpenShiftVersion))
 		}
+
 		cmdArgs = append(cmdArgs, "--"+key)
 		cmdArgs = append(cmdArgs, value)
 	}
 
+	// Check if extra flags are supported
+	for _, extraFlag := range extraFlags {
+		isFlag := strings.HasPrefix(extraFlag, "--")
+		key := strings.Trim(extraFlag, "-")
+
+		if !isFlag {
```

Comment:
> it is a first delivery...

@gbraad so what are you saying? Do you want to change this as part of this pull request with a follow up issue or do you want to fix is now?